app/src/main/java/org/groundplatform/android/ui/home/HomeScreenViewModel.kt

```diff
@@ -36,6 +37,8 @@ import org.groundplatform.android.ui.common.AbstractViewModel
 import org.groundplatform.android.ui.common.SharedViewModel
 import timber.log.Timber
 
+private const val WAITING_PHOTO_CAPTURE_KEY = "awaiting_photo_capture"
```

Comment:
Nit: WAITING vs awaiting - let's use "AWAITING" for consistency?

Reply:
Done!